web/html/src/manager/content-management/create-project/create-project.js

```diff
@@ -6,7 +6,7 @@ import TopPanelButtons from "./top-panel-buttons";
 import PropertiesCreate from "../shared/components/panels/properties/properties-create";
 import {showErrorToastr} from "components/toastr/toastr";
 import withPageWrapper from 'components/general/with-page-wrapper';
-import {hot} from 'react-hot-loader';
+import { hot } from 'react-hot-loader/root';
```

Comment:
`react-hot-loader` [docs](https://github.com/gaearon/react-hot-loader#getting-started) say: "Make sure react-hot-loader is required before react and react-dom". Since you're already touching all the imports, maybe you can apply this as well.

Reply:
@cbbayburt Good catch, I've updated it manually for now and I'll see if we can add linting to automatically cover this for the future.